### DISCUSSION TITLE
Upcoming Meeting Widget - Do Not Display

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#422] [Q&A] [answered]
There's got to be a quick answer to this. I hope so.

How can I stop displaying inactive meetings in the upcoming widget?

**Comment by tech2serve** [comment, marked as answer]
Duplicates #398

**Reply by Grateful-Viv** [reply]
I was hoping there could be some CSS that could be applied where attendance-inactive would not display at all. Right now choices are limited to either showing the upcoming meeting widget that displays everything (including inactive meetings) or putting the inactive meetings in draft mode. Ideally, would like to have folks select an option to see their inactive meetings as they're slowly reopening. In some cases, they've already opened but haven't told us.

**Reply by tech2serve** [reply]
PR #423, if merged and released, will remove inactive meetings from the widget.

This approach is better than trying to use CSS as the requested `count` argument would not be accurate.
